ui/src/app/pages/admin/admin-user.tsx

```diff
@@ -124,6 +112,31 @@ const EmailValidationErrorMessage = ({emailValidationResponse, updatedProfile, v
   return null;
 };
 
+const FreeCreditsUsage = ({aboveLimit, usage}) => {
```

Comment:
nit: type these? I don't know what rule we settled on for when we explicitly type stuff.

Reply:
I attempted to, unsuccessfully.  My assumption was that it was not possible.  But now Googling tells me otherwise, so I will make an attempt.

Reply:
typed now, with a `FreeCreditsProps`